dll/win32/t2embed/t2embed.spec

```diff
@@ -9,17 +9,17 @@
 @ stdcall TTIsEmbeddingEnabledForFacename(str ptr)
 @ stdcall TTLoadEmbeddedFont(ptr long ptr long ptr ptr ptr wstr str ptr)
 @ stdcall TTRunValidationTests(ptr ptr)
-@ stub _TTCharToUnicode@24
-@ stub _TTDeleteEmbeddedFont@12
-@ stdcall _TTEmbedFont@44(long long long ptr ptr ptr ptr ptr long long ptr) TTEmbedFont
-@ stub _TTEmbedFontFromFileA@52
-@ stub _TTEnableEmbeddingForFacename@8
-@ stub _TTGetEmbeddedFontInfo@28
-@ stdcall _TTGetEmbeddingType@8(long ptr) TTGetEmbeddingType
-@ stdcall _TTIsEmbeddingEnabled@8(long ptr) TTIsEmbeddingEnabled
-@ stdcall _TTIsEmbeddingEnabledForFacename@8(str ptr) TTIsEmbeddingEnabledForFacename
-@ stdcall _TTLoadEmbeddedFont@40(ptr long ptr long ptr ptr ptr wstr str ptr) TTLoadEmbeddedFont
-@ stub _TTRunValidationTests@8
+@ stdcall -i386 _TTCharToUnicode@24(ptr ptr long ptr long long) TTCharToUnicode
```

Comment:
Note: Forwarder argument lists could be omitted, couldn't they?

Reply:
In deed it does. Testing this I found a problem though: MSVC removes decorated exports as soon as there is an undecorated export of the same function. `_declspec(dllexport)` doesn't fix this either. Anyway, I'll remove the forwarders.

Reply:
Ah...
Now or later, you may want to check the following cases, for example, for potential issues:
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=%40%5B0-9%5D%2B%5C%28&sr=1